docs/vc-spec.md

```diff
@@ -223,6 +225,27 @@ After the user has successfully completed the flow, Internet Identity will respo
 
 An example of such a verifiable presentation can be found [here](https://www.w3.org/TR/vc-data-model/#example-jwt-payload-of-a-jwt-based-verifiable-presentation-non-normative).
 
+#### Receive an Error Message
+
+If the flow failed for any reason, Internet Identity returns an error.
+For privacy protection, the error does not give any details on the root cause of the failure.
+(This may change in the future, as some failures can be reported to the relying
+party without impacting user's privacy.)
+
+_Example_
+
+``` json
+{
+    "id": 1,
+    "jsonrpc": "2.0",
+    "error": {
```

Comment:
Not your mistake, but I realized that I copied the error definition from a broken example. We should fix that here too, and have a `code` instead of an `errorType`. See here: https://www.jsonrpc.org/specification#error_object

Reply:
Thanks for caching it, changed to `code`